indexer/services/ender/src/scripts/handlers/dydx_deleveraging_handler.sql

```diff
@@ -55,8 +56,9 @@ BEGIN
     */
     size = dydx_trim_scale(dydx_from_jsonlib_long(event_data->'fillAmount') *
                                  power(10, perpetual_market_record."atomicResolution")::numeric);
-    price = dydx_trim_scale(dydx_from_jsonlib_long(event_data->'price') *
+    quote_amount = dydx_trim_scale(dydx_from_jsonlib_long(event_data->'price') *
                                   power(10, QUOTE_CURRENCY_ATOMIC_RESOLUTION)::numeric);
+    price = dydx_trim_scale(quote_amount / size);
```

Comment:
nit: Add a comment indicating the event is incorrect and that's why this math is being done.

Reply:
I just updated the field name to totalQuoteQuantums as you suggested instead.